x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/external/request/openai/OpenAiUnifiedChatCompletionRequestEntityTests.java

```diff
@@ -702,124 +702,6 @@ public void testSerializationWithBooleanFields() throws IOException {
         assertJsonEquals(expectedJsonFalse, jsonStringFalse);
     }
 
-    // 9. Serialization with Missing Required Fields
```

Comment:
Can we add a test in the `UnifiedCompletionRequestTests` that parses a string without `content`?

https://github.com/elastic/elasticsearch/blob/467fdb879c6ed1ea084d02087a4afaa12babe7c6/x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/inference/action/UnifiedCompletionRequestTests.java


```
...

{
                "model": "gpt-4o",
                "messages": [
                  {
                    "role": "user",
                    ... (no content field)
...
```